packages/query/src/graphql/graphql.test.ts

```diff
@@ -143,7 +156,7 @@ describe('GraphqlModule', () => {
     `;
 
     const results = await server.executeOperation({query: GET_META});
-    expect(`${results.errors}`).toEqual(`Cannot query field "fakeMetadata" on type "_Metadata".`);
+    expect(`${results.errors} `).toEqual(`Cannot query field "fakeMetadata" on type "_Metadata". `);
```

Comment:
Can be reverted? Was this something done by the linter?

Reply:
This was probably changed by me accidentally.